sdk/python/feast/feature_store.py

```diff
@@ -998,6 +999,48 @@ def serve(self, port: int) -> None:
 
         feature_server.start_server(self, port)
 
+    @log_exceptions_and_usage
+    def upload_docker_image(self) -> None:
+        """Upload the docker image for the feature consumption server to the cloud."""
+
+        # TODO: add error checking and avoid hardcoding the region
+        repository_name = "feast-python-server-test"
+        feature_server_type = (
+            self.config.feature_server.type if self.config.feature_server else None
+        )
+        if feature_server_type == "aws_lambda":
+            import base64
+
+            import boto3
+            import docker
+            from botocore.exceptions import ClientError
+
+            docker_client = docker.from_env()
+            image_name = FEATURE_SERVER_IMAGE_FOR_TYPE[feature_server_type]
+            docker_client.images.pull(image_name)
+
+            ecr_client = boto3.client("ecr", region_name="us-west-2")
+            try:
+                ecr_client.create_repository(repositoryName=repository_name)
+            except ClientError:
+                pass
+            auth_token = ecr_client.get_authorization_token()["authorizationData"][0][
+                "authorizationToken"
+            ]
+            username, password = base64.b64decode(auth_token).decode("utf-8").split(":")
+
+            sts_client = boto3.client("sts")
+            aws_account = sts_client.get_caller_identity()["Account"]
+            ecr_address = f"{aws_account}.dkr.ecr.us-west-2.amazonaws.com"
+            docker_client.login(
+                username=username, password=password, registry=ecr_address
+            )
+
+            # Pushing will likely take several minutes.
+            image = docker_client.images.get(image_name)
+            image.tag(f"{ecr_address}/{repository_name}:latest")
```

Comment:
`:latest` could cause issues. Any system that has already pulled the `latest` image may skip pulling an image again. Can we implement some kind of versioning?

Reply:
How about we use `feast.__version__` as a tag?